CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ If you want more details about available licensing or the contribution agreement
 
 ## Contributing to the Shopware code base
 If you want to learn how to contribute code to Shopware, please refer to [Contributing Code](https://docs.shopware.com/en/shopware-platform-dev-en/contribution/contributing-code?category=shopware-platform-dev-en/contribution).
-Also make sure that you add a changelog file which describes your changes in a meaningful way. For more information refer to [this document](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md#workflow).
+Also make sure that you add a changelog file which describes your changes in a meaningful way. For more information refer to [this document](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md).
```

Comment:
Thanks for good catch, @kiplingi 

I'm rather greedy :) We have similar one in `src/Docs/Resources/current/70-contribution/10-contribution-guideline.md`. You don't mind if you help to fix it as well in a single MR, then we could provide a proper documentation for people?

Thanks in advance.

Reply:
isn't the `src/Docs` deprecated anyway?

Reply:
Yes you can ignore it

Reply:
Sounds mysterious ;) Is the doc moving, or do we get a new one?

Reply:
the docs are located here for a while now :slightly_smiling_face: https://github.com/shopware/docs